scripts/daq_utils.py

```diff
@@ -3,232 +3,197 @@
 #
 # Do not use non-standard python.
 ####################################################
+import errno
 import getpass
-import logging
 import os
 import socket
 import subprocess
-import sys
 import time
 from subprocess import PIPE
 
-DAQMGR_HUTCHES = ["tmo", "rix", "txi", "mfx"]
+DAQMGR_HUTCHES = ["tmo", "rix", "txi", "ued"]
 LOCALHOST = socket.gethostname()
 SLURM_PARTITION = "drpq"
 SLURM_JOBNAME = "submit_daq"
+SLURM_SCRIPT = "submit_daq.sh"
 SLURM_OUTPUT = "slurm_daq.log"
 MAX_RETRIES = 10
 
 
-def call_subprocess(*args):
+def silentremove(filename):
     try:
-        output = subprocess.check_output(args, stderr=PIPE).strip()
-        return output.decode("utf-8")
-    except subprocess.CalledProcessError as e:
-        logging.error("Subprocess call '%s' failed with error: %s", " ".join(args), e)
-        raise
+        os.remove(filename)
+    except OSError as e:
+        if e.errno != errno.ENOENT:  # errno.ENOENT = no such file or directory
+            raise
+
+
+def call_subprocess(*args):
+    return str(subprocess.check_output(args, stderr=PIPE).strip(), "utf-8")
 
 
 def call_sbatch(cmd, nodelist, scripts_dir):
     sb_script = "#!/bin/bash\n"
-    sb_script += f"#SBATCH --partition={SLURM_PARTITION}\n"
-    sb_script += f"#SBATCH --job-name={SLURM_JOBNAME}\n"
-    sb_script += f"#SBATCH --nodelist={nodelist}\n"
-    sb_script += f"#SBATCH --output={os.path.join(scripts_dir, SLURM_OUTPUT)}\n"
+    sb_script += f"#SBATCH --partition={SLURM_PARTITION}" + "\n"
+    sb_script += f"#SBATCH --job-name={SLURM_JOBNAME}" + "\n"
+    sb_script += f"#SBATCH --nodelist={nodelist}" + "\n"
+    sb_script += f"#SBATCH --output={os.path.join(scripts_dir, SLURM_OUTPUT)}" + "\n"
```

Comment:
It's at least a bit strange to pull the `"\n"` out of the fstrings. The resulting string is the same but there's an extra step now.